### DISCUSSION TITLE
klipperscreen shows console but will not function after update today

### DISCUSSION THREAD

**Discussion by Riggaplz0451** [KlipperScreen/KlipperScreen#926] [Q&A]
so 2 days ago i conducted the normal updates to my voron 2.4 after which my tft50 klipper screen went into consul mode nd wont come out of it. i have reinstalled KS twice now both times with no wins. 
i have a few photo's to help better describe the situation. 
![334736385_156537950599898_1150995733072061229_n](https://user-images.githubusercontent.com/119629826/224525331-806eba51-2873-4c59-8cc6-df621e259c43.png)
![332839509_1157317324945865_7387358987261194046_n](https://user-images.githubusercontent.com/119629826/224525334-459d372a-790e-4662-a469-c3f3652be991.jpg)

**Comment by retronexus** [comment]
I have the same issue, after updating the raspberry yesterday, the touchscreen does not work anymore

**Comment by Tonybarrett62** [comment]
Added KlipperScreen (Using Kiauh) on an existing Mainsail OS (installed using the Raspiberry pi Imager), I also ran into this problem.

Checking the logs
 KlipperScreen - no screen found

Xorg @ /var/log/Xorg.0.log, 
 FBturbo "module does not exist" warning.

Further searching lead me to this Stack Exchange question - https://raspberrypi.stackexchange.com/questions/114383/xorg-doesnt-start-after-upgrade-from-stretch-to-buster

Which lead me to this command.
sudo mv /usr/share/X11/xorg.conf.d/99-fbturbo.conf ~

Took about 5 seconds for the display to show the Klipper screen. It has so far persisted with reboots. 
Appears to be a bug from OS updates.

**Reply by Riggaplz0451** [reply]
whoot whoot. this is great but it has not persisted through reboots. the command itself does turn my screen back on but then after a reboot its right back to terminal.

**Reply by ThePullOutCouch** [reply]
Same issue, fixes the screen, even makes the touch work which I was having issues with but does not persist through a reboot.

**Reply by Tonybarrett62** [reply]
Login prompt or terminal?

I have also enabled auto login @ raspi-config

**Reply by Riggaplz0451** [reply]
![received_1157317318279199.jpeg](https://user-images.githubusercontent.com/119629826/229269093-e545a42c-e418-40b4-92a0-dd485fa9eddd.jpeg)

**Reply by Tonybarrett62** [reply]
What does it say in the logs? No screen found or something else?

**Reply by Riggaplz0451** [reply]
> What does it say in the logs? No screen found or something else?
> 

Yes sir.

**Reply by KennyBeaudoin** [reply]
i have the exact same issue on one of my printer ... whats going on with the suport and bugs and when will it be fixed ? thanks

**Comment by alfrix** [comment]
let's keep this in one place #971
